ingest/processors/corelight.go

```diff
@@ -162,7 +154,10 @@ func (c *Corelight) process(mp map[string]interface{}, og []byte) (tag string, t
 	} else if line, ok = emitLine(ts, headers, mp); !ok {
 		tag = defaultTag
 		line = og
+	} else {
+		line = bytes.TrimRight(line, "-\t\n")
```

Comment:
Won't this trim off the last - column? Surely we want to keep that, if only for consistency's sake with files ingested natively as TSV

Reply:
fixed.